gtda/homology/point_clouds.py

```diff
@@ -457,3 +457,194 @@ def transform(self, X, y=None):
         Xt = np.stack(Xt)
         Xt = np.nan_to_num(Xt, posinf=self.infinity_values_)
         return Xt
+
+
+@adapt_fit_transform_docs
+class EuclideanCechPersistence(BaseEstimator, TransformerMixin):
+    """`Persistence diagrams <https://giotto.ai/theory>`_ resulting from
+    `Cech filtrations <https://giotto.ai/theory>`_.
+    Given a `point cloud <https://giotto.ai/theory>`_ in Euclidean space,
+    information about the appearance and disappearance of topological
+    features (technically, `homology classes <https://giotto.ai/theory>`_) of
+    various dimensions and at different scales is summarised in the
+    corresponding persistence diagram.
+
+    Parameters
+    ----------
+    homology_dimensions : iterable, optional, default: ``(0, 1)``
+        Dimensions (non-negative integers) of the topological features to be
+        detected.
+
+    coeff : int prime, optional, default: ``2``
+        Compute homology with coefficients in the prime field
+        :math:`\\mathbb{F}_p = \\{ 0, \\ldots, p - 1 \\}` where
+        :math:`p` equals `coeff`.
+
+    max_edge_length : float, optional, default: ``numpy.inf``
+        Upper bound on the maximum value of the Vietoris-Rips filtration
+        parameter. Points whose distance is greater than this value will
+        never be connected by an edge, and topological features at scales
+        larger than this value will not be detected.
+
+    infinity_values : float or None, default : ``None``
+        Which death value to assign to features which are still alive at
+        filtration value `max_edge_length`. ``None`` has the same behaviour
```

Comment:
We introduced a rewording for this. In `VietorisRipsPersistence`, we say "``None`` means that this death
        value is declared to be equal to `max_edge_length`."